heat/core/dndarray.py

```diff
@@ -483,6 +484,131 @@ def astype(self, dtype, copy=True):
 
         return self
 
+    def balance_(self):
+        """
+        Function for balancing a DNDarray between all nodes. To determine if this is needed use the is_balanced function.
+        If the DNDarray is already balanced this function will do nothing. This function modifies the DNDarray itself and will not return anything.
+
+        Examples
+        --------
+        >>> a = ht.zeros((10, 2), split=0)
+        >>> a[:, 0] = ht.arange(10)
+        >>> b = a[3:]
+        [0/2] tensor([[3., 0.],
+        [1/2] tensor([[4., 0.],
+                      [5., 0.],
+                      [6., 0.]])
+        [2/2] tensor([[7., 0.],
+                      [8., 0.],
+                      [9., 0.]])
+        >>> b.balance()
+        >>> print(b.gshape, b.lshape)
+        [0/2] (7, 2) (1, 2)
```

Comment:
i am showing it with 3 processes. if you run it with -np 3 this is what you get. if you look at the print above it it shows what the data actually looks like

Reply:
But you balanced it which resulted in the tensor at rank = 0 to have a dimension of (3, 2), which is shown in the print below? Otherwise the dimensions wont add up to the total of 7 of the global shape.

Reply:
you are correct. i have fixed this